packages/valory/skills/market_manager_abci/graph_tooling/requests.py

```diff
@@ -184,7 +187,9 @@ def _fetch_redeem_info(self) -> Generator[None, None, Optional[list]]:
         self._fetch_status = FetchStatus.IN_PROGRESS
 
         safe = self.synchronized_data.safe_contract_address
-        query = trades.substitute(creator=safe.lower())
+        redeem_margin = self.params.redeem_margin_days * DAY_UNIX
+        from_timestamp = self.synced_time - redeem_margin
+        query = trades.substitute(creator=safe.lower(), from_timestamp=from_timestamp)
```

Comment:
I assume the event filter later on is made s.t. the lowest block is used from this response, correct?
I am asking because I don't see changes related to `from_block`, or something like that.

Reply:
Yeap, exactly.